frontend/public/components/utils/kebab.tsx

```diff
@@ -129,6 +129,15 @@ const kebabFactory: KebabFactory = {
     }),
     accessReview: asAccessReview(kind, obj, 'patch'),
   }),
+  RemoveVolume: (kind, obj, volume) => ({
+    label: 'Remove Volume',
+    callback: () => removeVolumeModal({
+      kind,
+      resource: obj,
+      volume,
+    }),
+    accessReview: asAccessReview(kind, obj, 'patch'),
```

Comment:
This should be `'update'` as written, but it might be better to convert to patch.